index.js

```diff
@@ -9,8 +10,20 @@ const keyRegex = (str) => {
   return new RegExp('^' + str.replace('*','.*'))
 }
 
+let observer;
+
+const observable = new Observable((_observer) => {
+  observer = _observer;
+  return () => {}
+});
```

Comment:
I agree, a bit unsure about it tho. I can look more into it, any tips would be appreciated.